pytorch_lightning/trainer/connectors/data_connector.py

```diff
@@ -127,16 +127,25 @@ def on_trainer_init(
         self.trainer._is_data_prepared = False
 
     def _select_data_fetcher(self) -> AbstractDataFetcher:
+        running_stage_to_step_method_name = {
+            RunningStage.TRAINING: "training_step",
+            RunningStage.SANITY_CHECKING: "validation_step",
+            RunningStage.VALIDATING: "validation_step",
```

Comment:
Should we include predict ?

Reply:
Good point. I want to add support to predict in a separate PR to minimize the changes here

Reply:
Good :)